internal/push.go

```diff
@@ -92,7 +92,9 @@ var loadBundle = func(ctx context.Context, memoryStore *content.Memorystore, pat
 }
 
 var loadConfig = func(memoryStore *content.Memorystore) ocispec.Descriptor {
-	return memoryStore.Add("config.json", customConfigMediaType, []byte(configContents))
+	descriptor := memoryStore.Add("config.json", customConfigMediaType, []byte(configContents))
+	descriptor.Annotations = nil
```

Comment:
The `push` was somehow broken for Elastic if the annotations were not set to nil. This change also implies that we're not printing to the user that we're pushing the `config.json` file to a registry, which is less confusing.